frontend-react/src/utils/GetIsIE.ts

```diff
@@ -0,0 +1,4 @@
+export function getIsIE() {
```

Comment:
Rather than creating a utility function strictly for detecting Internet Explorer, I wonder if this could be broadened to `(d|D)evice.ts` where we could eventually house functionality to detect other browsers and/or mobile devices as well.  And maybe even a `getBrowser` convenience function down the line.  Whatchu think?

Also, I think we can hoist the IE regex out since we're not using the global flag:

```tsx
const IE_REGEX = /Trident\/|MSIE/;

export function getIsIE() {
    return IE_REGEX.test(window.navigator.userAgent);
}
```

Reply:
I feel at that rate you've broadened back to needing a library that keeps up to date with devices.

Reply:
That's a fair point, though I'd argue that with that file it'd be more gradual and opt-in rather than having an entire library with 99% of it unused.

In any case, I feel like it's unnecessary to have an entire file devoted to checking against a single regular expression.  Not a blocker, but it seems unnecessary 🤷 

(Also, obligatory "mind adding some tests?" comment  😆)

Reply:
With IE behind us we should be programmatically making choices based on available features (which should be getting polyfilled anyways) and device hardware capabilities (screen size, dpi, etc.) and not necessarily the device/browser itself.

Reply:
Sure, maybe device.ts is a misnomer, but "feature" is a very overloaded term -- my thought here was that we'd have a utility file for any sort of the aforementioned detection going forward.  But again, not a blocker at this point.